### PR TITLE
Improve accuracy a little bit

### PR DESCRIPTION
**Pull request by ghost**
I didn't jump _all_ the way into using Meeus' approximations, but I did add a term of accuracy by synthesizing a few sources. Also more tests and minor optimizations are nice.

### CODE REVIEW

File: lib/solar.js [hunk 1]
```diff
@@ -18,19 +23,24 @@ function _mean_anomaly(t) {
...
-const L1 = 0.020 * RAD;
+const L0 = 1.9148 * RAD;
+const L1 = 0.0200 * RAD;
+const L2 = 0.0003 * RAD;
 function _longitude(t) {
   const g = _mean_anomaly(t);
-  return Q0 + Q1 * t + L0 * Math.sin(g) + L1 * Math.sin(2 * g);
+  const sin_g = Math.sin(g);
+  const sin_2g = Math.sin(2 * g);
+  const sin_3g = (3 - 4 * sin_g * sin_g) * sin_g;
```
Comment (jareyes) [inline lib/solar.js]: This is wizardry.

Comment (nonrenewable) [reply lib/solar.js]: I'm pretty sure i never studied these... or maybe like 1 month ever. i looked it up though and yep, sure enough that's how you do triple angle identity! 💥

File: lib/solar.js [hunk 2]
```diff
@@ -18,19 +23,24 @@ function _mean_anomaly(t) {
...
+  return Q0 + Q1 * t + L0 * sin_g + L1 * sin_2g + L2 * sin_3g;
 }
 
 // derivative of ecliptic longitude
 function _longitude_prime(t) {
   const g = _mean_anomaly(t);
   const cos_g = Math.cos(g);
   const cos_2g = 2 * cos_g * cos_g - 1;
-  return Q1 + G1 * (L0 * cos_g + 2 * L1 * cos_2g);
+  const cos_3g = 2 * cos_g * cos_2g - cos_g;
```
Comment (nonrenewable) [inline lib/solar.js]: ok this one was sneaky but i'm sold.

**Review by nonrenewable-zz — APPROVED** [review]
legit trig-ing!

File: lib/solar.js (hunk 1 above)
Comment (ironwallaby) [reply lib/solar.js]: You owe it to yourself to keep the recurrence formulae for the multiple angle formulas on your wall!

**Pull request merged**